code/mouse.py

```diff
@@ -148,20 +148,13 @@ def mouse_trigger_zoom_mouse():
         if eye_zoom_mouse.zoom_mouse.enabled:
             eye_zoom_mouse.zoom_mouse.on_pop(eye_zoom_mouse.zoom_mouse.state)
 
-    def mouse_drag():
+    def mouse_drag(button: int):
         """(TEMPORARY) Press and hold/release button 0 depending on state for dragging"""
-        # todo: fixme temporary fix for drag command
-        button_down = len(list(ctrl.mouse_buttons_down())) > 0
-        # print(str(ctrl.mouse_buttons_down()))
-        if not button_down:
-            # print("start drag...")
-            ctrl.mouse_click(button=0, down=True)
-            # app.notify("drag started")
-        else:
-            # print("end drag...")
-            ctrl.mouse_click(button=0, up=True)
+        ctrl.mouse_click(button=button, down=True)
 
-        # app.notify("drag stopped")
+    def end_mouse_drag(button: int):
```

Comment:
I'd say stick with the "mouse_{name here}" convention, so mouse_drag_end would be preferable.